components/dashboard/src/components/Modal.tsx

```diff
@@ -14,8 +15,8 @@ export default function Modal(props: {
     specify?: string;
     title?: string;
     hideDivider?: boolean;
-    buttons?: React.ReactChild[] | React.ReactChild;
-    children: React.ReactChild[] | React.ReactChild;
+    buttons?: ReactNode;
```

Comment:
`ReactNode` type encompasses arrays or single children, but also allows for `null`.